docs/coordinates/galactocentric.rst

```diff
@@ -0,0 +1,108 @@
+.. _coordinates-galactocentric:
+
+==========================================
+Transforming to Galactocentric coordinates
+==========================================
+
+[sec:transform]
```

Comment:
Hm, will look in to that -- I used pandoc to convert from Latex to ReST and I guess I didn't double check that everything was sensible afterwards :)